packages/gap-web-ui/src/components/question-page/inputs/Checkboxes.tsx

```diff
@@ -73,6 +74,86 @@ const Checkboxes = ({
+                if (!divideLastCheckboxOption) return undefined;
+                if (divideCheckboxIndex === 1 && index === 0)
+                  return 'exclusive';
+                if (
+                  divideCheckboxIndex === disabledOptions.length - 1 &&
+                  index === disabledOptions.length - 1
+                )
+                  return 'exclusive';
+                return undefined;
+              };
+
+              return (
+                <React.Fragment key={value}>
+                  {divideLastCheckboxOption &&
+                    index === divideCheckboxIndex && (
+                      <div className="govuk-checkboxes__divider">or</div>
+                    )}
+
+                  <div className="govuk-checkboxes__item">
+                    <input
+                      className="govuk-checkboxes__input"
+                      id={index === 0 ? fieldName : `${fieldName}-${index + 1}`}
+                      name={fieldName}
+                      type="checkbox"
+                      value={value}
+                      defaultChecked={
+                        defaultCheckboxes &&
+                        defaultCheckboxes.indexOf(value) >= 0
+                      }
+                      disabled={true}
+                      data-behaviour={getDataBehaviour()}
+                      data-cy={`cy-checkbox-value-${value}`}
+                      data-aria-controls={
+                        typeof option === 'object' && option.conditionalInput
+                          ? `conditional-${
+                              index === 0
+                                ? fieldName
+                                : `${fieldName}-${index + 1}`
+                            }`
+                          : undefined
+                      }
+                    />
+                    <label
+                      className="govuk-label govuk-checkboxes__label"
+                      htmlFor={
+                        index === 0 ? fieldName : `${fieldName}-${index + 1}`
+                      }
+                      data-cy={`cy-checkbox-label-${value}`}
+                    >
+                      {typeof option === 'string' ? value : option.label}
+                    </label>
+                  </div>
+
+                  {typeof option === 'object' && option.conditionalInput && (
+                    <div
+                      className="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden"
+                      id={`conditional-${
+                        index === 0 ? fieldName : `${fieldName}-${index + 1}`
+                      }`}
+                      data-testid={`conditional-input-wrapper-${index + 1}`}
+                    >
+                      <div className="govuk-form-group">
+                        {option.conditionalInput}
+                      </div>
+                    </div>
+                  )}
+                </React.Fragment>
+              );
+            })}
```

Comment:
I think we can improve the interface here and reduce duplication but need to have a think about it

Reply:
One of the issues I think we'd have with the approach here is that it's harder to implement this in other places as we have to partition the array of checkboxes into disabled and not, and it could cause UI issues as all the disabled checkboxes will appear at the top and non-disabled will appear at the bottom as a result.

To fix this we can follow a similar interface as we have with defaultCheckboxes.

We can continue to have the `disabled` prop which disables all of them, but then additionally have a `disabledCheckboxes` prop which takes an array of checkbox keys (just like with defaultCheckboxes) and then use that to figure out which of the checkboxes should be disabled, if any. We should still prioritise the `disabled` prop for disabling all, of course.

Once this is done we should also update the tests and stories for the checkboxes to ensure it stays properly documented.

Happy to chat about it if you'd like :)

Reply:
Oh and by doing this we can remove this entire block of code too - and just update the disabled prop in the `options` mapping :)